update-bootengine

```diff
@@ -104,6 +104,13 @@ else
     DRACUT_ARGS+=( "--no-kernel" )
 fi
 
+if [ -f /.dockerenv ] ; then
+    # don't try to copy XATTR in overlayfs
+    echo "Running in a Docker container, not copying xattrs."
+    DRACUT_NO_XATTR=1
+    export DRACUT_NO_XATTR
```

Comment:
Out of curiosity, is there any difference between this and the following oneliner:
```suggestion
    export DRACUT_NO_XATTR=1
```